locksmith/src/routes/v2/metadata.ts

```diff
@@ -50,4 +51,10 @@ router.put(
   updateUserMetadata
 )
 
+router.get(
+  '/:network/locks/:lockAddress/users/:userAddress',
+  authenticatedMiddleware,
```

Comment:
Iirc this middleware onky checks that there's a connected user, not that the user is a special one